daemon/mgr/image.go

```diff
@@ -594,7 +571,7 @@ func (mgr *ImageManager) addReferenceIntoStore(id digest.Digest, ref reference.N
 	return nil
 }
 
-func (mgr *ImageManager) containerdImageToImageInfo(ctx context.Context, id digest.Digest) (types.ImageInfo, error) {
+func (mgr *ImageManager) containerdImageToImageInfo(ctx context.Context, id digest.Digest, referenceFilters []string) (types.ImageInfo, error) {
```

Comment:
IMO, `containerdImageToImageInfo` is used to get the image from containerd and converts it to `types.ImageInfo`, it is not suitable for doing filter.

Reply:
I prefer to do filtering in `ListImages` and redefining a new method to filter multiple tags.

Reply:
But if you redefine a func to do that, you'll find that most of the logic between the two funcs are same. I think it's redundant.